docs/guide/src/example-plugin/plugin.go

```diff
@@ -69,12 +69,12 @@ func (ep *ExamplePlugin) RunTx(store types.KVStore, ctx types.CallContext, txByt
 	return abci.OK
 }
 
-func (ep *ExamplePlugin) InitChain(store types.KVStore, vals []*abci.Validator) {
+func (cp *ExamplePlugin) InitChain(store types.KVStore, vals []*abci.Validator) {
```

Comment:
`cp` ? let's keep it `ep` for this example plugin. Thanks for fixing the methods though